test/Dialect/SV/canonicalization.mlir

```diff
@@ -1,46 +1,49 @@
 // RUN: circt-opt -canonicalize='top-down=true region-simplify=true' %s | FileCheck %s
 
 // CHECK-LABEL: func @if_dead_condition(%arg0: i1) {
+// CHECK-NEXT:    %0 = sv.fd stderr
```

Comment:
Mega nit: For any test which isn't already heavily using `%0`, `%1`, `%2`, etc., I'd capture this in a `%[[STDERR:.+]] =`.